libtest/src/test/java/org/jetbrains/kotlinx/lincheck/AnalyzeAccessor.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.kotlinx.lincheck
+
+import org.jetbrains.kotlinx.lincheck.strategy.IterationResult
+
+internal fun linCheckAnalyze(clazz: Class<*>, options: Options<*, *>?): Map<CTestConfiguration, List<IterationResult>> = LinChecker.analyze(clazz, options)
```

Comment:
It is better to make the method public and add a comment that this is an internal API. Don't want to create a new file for solving this issue, it makes the code more complicated

Reply:
It seems to me that the fact that autocompletion suggests `checkDetailed` when typing `check` will attract unwanted attention